packages/block-library/src/template-part/editor.scss

```
@@ -23,8 +23,10 @@
 	z-index: z-index(".block-library-template-part__selection-search");
 }
 
-.is-outline-mode .block-editor-block-list__block:not(.remove-outline).wp-block-template-part,
-.is-outline-mode .block-editor-block-list__block:not(.remove-outline).is-reusable {
+// Removed .is-outline-mode so colors take effect properly in the block editor.
```

Is this a comment that makes sense without seeing the diff? It seems hard to understand what that class is all about? Perhaps say something like 
```suggestion
// We don't use .is-outline-mode in this case so colors take effect properly in the block editor.
```